client/src/reducers/feedReducers.js

```diff
@@ -0,0 +1,41 @@
+import {
+  ADD_OPTION,
+  REMOVE_OPTION,
+  REMOVE_ALL_OPTIONS,
+  TOGGLE_STATE,
+  SET_VALUE,
+} from "../actions/feedActions";
+
+export const feedReducer = (oldState, action) => {
+  let newState = Object.assign({}, oldState);
+  const { type, key, value } = action;
+  switch (type) {
+    case TOGGLE_STATE:
+      newState[key] = !newState[key];
```

Comment:
I would probably write it a little shorter too like `return { ...state, [key]: !state[key] };`